apps/meteor/server/settings/push.ts

```diff
@@ -104,13 +109,36 @@ export const createPushSettings = () =>
 			});
 			await this.add('Push_gcm_api_key', '', {
```

Comment:
I mean I think we could just keep it to avoid confusion, since the change is going to be kind of big already
and when the API reaches end of life we just remove it + sends some emails so people update their settings